build/inverted-index-helper.js

```diff
@@ -5,37 +5,70 @@ var _createClass = function () { function defineProperties(target, props) { for
 
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
-var invertedIndexHelper = function () {
-  function invertedIndexHelper() {
-    _classCallCheck(this, invertedIndexHelper);
+/**
```

Comment:
### Critical

Generated files should not be in version control. If it can be generated, it should be ignored.